### PR TITLE
Plans: expanded nudge abtest

### LINKED ISSUE #7777: ABTest: replace nudges with ExpandedNudge
#7672 introduced ExpandedNudge component.

We are basing our assumptions on a premise that expanded nudge is better than regular one. But we didnt really test it. It would be nice to uniformly test nudges in regular / expanded version everywhere.

Nudges to test in BOTH versions:

## Themes

zrzut ekranu 2016-08-31 o 09 51 00

## Settings

zrzut ekranu 2016-08-31 o 09 51 24

## Stats

zrzut ekranu 2016-08-31 o 09 52 01

## Media

zrzut ekranu 2016-08-31 o 09 52 39

## Questions:

Should we include nudges introduced in #7648, #7651, #7647 in "small" versions as a part of the same test?

CC @rralian @apeatling  - please advise

### PR DESCRIPTION
**Pull request by artpi**
Resolves #7777

Introduces ExpandedNudge vs Regular nudge ABtest in:

- SEO settings
- Google Analytics from #7747
- Video tab in media

## Testing

- Use Free site
- Assign yourself `expandedNudge`: `regular`ABtest
- Go to `/settings/seo/:site`
- See regular nudge
- Go to `/settings/analytics/:site`
- See regular nudge
- Assign yourself `expandedNudge`: `expanded`ABtest
- Go to `/settings/seo/:site`
- See expanded nudge
- Go to `/settings/analytics/:site`
- See expanded nudge
- Assign yourself `expandedNudge`: `expanded`ABtest
- Create post, open "Video" tab in media modal
- See expanded nudge
- Assign yourself `expandedNudge`: `regular`ABtest
- Create post, open "Video" tab in media modal
- See regular nudge

CC @vindl @rralian @apeatling @gwwar @mtias 

Test live: https://calypso.live/?branch=update/expanded-nudge-abtest

### CODE REVIEW

File: client/my-sites/site-settings/form-analytics.jsx [hunk 1]
```diff
@@ -201,28 +202,40 @@ export default React.createClass( {
...
-					this.translate(
-						'Learn what people are looking for and what they like with In-Page Analytics. ' +
-						'Then tailor your marketing and site content for maximum impact.'
-					)
-				] }
-			/>
-		);
+		if( abtest( 'expandedNudge' ) === 'expanded' ) {
+			return (
+				<GoogleAnalyticsUpgradeNudge
```
Comment (vindl) [inline client/my-sites/site-settings/form-analytics.jsx]: Should we rename this component to `UpgradeNudgeExpanded`? We could also do it in a janitorial PR that would cover other nudges too.

**Comment by vindl** [comment]
I've modified my PR in order to include VideoPress nudge in this test too (#7839). 

Tested it and it works as expected. This looks good to me. :shipit:

File: client/lib/abtest/active-tests.js [hunk 2]
```diff
@@ -33,6 +33,15 @@ module.exports = {
 		},
 		defaultVariation: 'clickableButton'
 	},
+	expandedNudge: {
+		datestamp: '20160904',
```
Comment (rralian) [inline client/lib/abtest/active-tests.js]: don't forget to change the date before merging

**Comment by rralian** [comment]
Looks good to me. I say :shipit: once the circleci build is happy _and after updating the test date_. I just triggered circleci to run the tests again.

File: client/lib/abtest/active-tests.js (hunk 2 above)
Comment (artpi) [inline client/lib/abtest/active-tests.js]: Done!

**Pull request merged**